app/app.go

```diff
@@ -884,13 +909,6 @@ func NewApplication(
 	return app
 }
 
-func (app *Application) CreateUpgradeHandler(mm *module.Manager, configurator module.Configurator) upgradetypes.UpgradeHandler {
```

Comment:
Seems that this change is not related to wasmbindings

Reply:
Yeah, it is not. It was an unused function.